content/en/docs/security/_index.md

```diff
@@ -158,4 +155,167 @@ At a minimum, the [Pod Security Standards](/policies/pod-security/) and [best pr
+
+**Mitigation**
+
+* [Mitigation ID 4 - Webhook uses TLS encryption for all traffic](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#mitigation-id-4---webhook-uses-tls-encryption-for-all-traffic)
+
+  Kyverno uses HTTPS for all webhook traffic.
+
+### [Threat ID 8 - Attacker carries out a MITM attack on the webhook](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#threat-id-8---attacker-carries-out-a-mitm-attack-on-the-webhook)
+
+**Mitigation**
+
+* [Mitigation ID 5 - Webhook mutual TLS authentication is used](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#mitigation-id-5---webhook-mutual-tls-authentication-is-used)
+
+     By default, Kyverno generates a CA and X.509 certificates for the webhook registration. As custom CA and certificates can be used as discussed in the [installation guide](/docs/installation/#option-2-use-your-own-ca-signed-certificate).  
+
+
+### [Threat ID 9 - Attacker steals traffic from the webhook via spoofing](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#threat-id-9---attacker-steals-traffic-from-the-webhook-via-spoofing)
+
+
+**Mitigation**
+
+* [Mitigation ID 5 - Webhook mutual TLS authentication is used](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#mitigation-id-5---webhook-mutual-tls-authentication-is-used)
+
+     By default, Kyverno generates a CA and X.509 certificates for the webhook registration. As custom CA and certificates can be used as discussed in the [installation guide](/docs/installation/#option-2-use-your-own-ca-signed-certificate).  
+
+### [Threat ID 10 - Abusing a mutation rule to create a privileged container](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#threat-id-10---abusing-a-mutation-rule-to-create-a-privileged-container)
+
+**Mitigation**
+
+* [Mitigation ID 6 - All rules are reviewed and tested](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/admission-control/kubernetes-admission-control-threat-model.md#Mitigation-id-6---all-rules-are-reviewed-and-tested)
+
+     Kyverno rules are Kubernetes resources written in YAML and managed by a declarative OpenAPIv3 schema. This makes it easy understand  policy and to apply `policy as code` practices, such as code reviews, to Kyverno policies.
```

Comment:
"This makes it easy to understand policy and to apply policy as code practices, such as code reviews, to Kyverno policies." (insert "to", delete space between "understand" and "policy", remove code formatting from "policy as code" phrase)